doc/user/bgp.rst

```diff
@@ -3877,6 +3877,123 @@ Example of how to set up a 6-Bone connection.
+
+CLI Configuration:
+------------------
+Below configuration can be done in router bgp mode and allows the user to configure the tcp-mss value per neighbor. The configuration gets applied only after hard reset is performed on that neighbor. If we configure tcp-mss on both the neighbors then both neighbors need to be reset.
+The configuration takes effect based on below rules, so there is a configured tcp-mss and a synched tcp-mss value per TCP session.
+By default if the configuration is not done then the TCP max segment size is set to the Maximum Transmission unit (MTU) – (IP/IP6 header size + TCP header size + ethernet header). For IPv4 its MTU – (20bytes IP header + 20 bytes TCP header + 12bytes ethernet header) and for IPv6 its MTU – (40bytes IPv6 header + 20 bytes TCP header + 12 bytes ethernet header).
+If the config is done then it reduces 12-14 bytes for the ether header and uses it after synchronizing in TCP handshake.
+
+.. clicmd:: [no] neighbor <A.B.C.D|X:X::X:X|WORD> tcp-mss (1-65535)
+
+When tcp-mss is configured kernel reduces 12-14 bytes for ethernet header. E.g. if tcp-mss is configured as 150 the synched value will be 138.
+
+Note: configured and synched value is different since TCP module will reduce 12bytes for ethernet header.
+
+Running config:
+---------------
+
+.. code-block:: frr
+
+    frr# show running-config 
+    Building configuration...
+    
+    Current configuration:
+    !
+    router bgp 100
+     bgp router-id 1.1.1.1
+     neighbor 10.1.1.2 remote-as 100
+     neighbor 10.1.1.2 tcp-mss 150       => new entry
+     neighbor 2000::2 remote-as 100
+     neighbor 2000::2 tcp-mss 400        => new entry
+
+Show command:
+-------------
+
+.. code-block:: frr
+
+    frr# show bgp neighbors 10.1.1.2 
+    BGP neighbor is 10.1.1.2, remote AS 100, local AS 100, internal link
+    Hostname: frr
+      BGP version 4, remote router ID 2.2.2.2, local router ID 1.1.1.1
+      BGP state = Established, up for 02:15:28
+      Last read 00:00:28, Last write 00:00:28
+      Hold time is 180, keepalive interval is 60 seconds
+      Configured tcp-mss is 150, synced tcp-mss is 138     => new display
+
+.. code-block:: frr
+
+    frr# show bgp neighbors 2000::2 
+    BGP neighbor is 2000::2, remote AS 100, local AS 100, internal link
+    Hostname: frr
+      BGP version 4, remote router ID 2.2.2.2, local router ID 1.1.1.1
+      BGP state = Established, up for 02:16:34
+      Last read 00:00:34, Last write 00:00:34
+      Hold time is 180, keepalive interval is 60 seconds
+      Configured tcp-mss is 400, synced tcp-mss is 388      => new display
+
+
+Show command json output:
+-------------------------
+
+.. code-block:: frr
+
+    frr# show bgp neighbors 2000::2 json 
+    {
+      "2000::2":{
+        "remoteAs":100,
+        "localAs":100,
+        "nbrInternalLink":true,
+        "hostname":"frr",
+        "bgpVersion":4,
+        "remoteRouterId":"2.2.2.2",
+        "localRouterId":"1.1.1.1",
```

Comment:
Please use a prefix designed for documentation, no Cloudflare here.

Reply:
Done